src/chrome/content/rules/transfermarkt.de.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+		404:
+			transfermarkt.de
+-->
+<ruleset name="Transfermarkt.de">
+
+	<target host="transfermarkt.de" />
+	<target host="*.transfermarkt.de" />
+		<test url="http://www.transfermarkt.de/" />
+		<test url="http://www.transfermarkt.de/live/counter" />
+		<test url="http://www.transfermarkt.de/navigation/hauptnavi" />
```

Comment:
The idea of the 3 test urls requirement for wildcards is to have a bunch of subdomains covered. If `www` is the only subdomain you should avoid the wildcard target.

Reply:
Please add a hint about the wildcard DNS record in the comments.